internal/apis/certmanager/types_certificate.go

```diff
@@ -84,6 +84,9 @@ type CertificateSpec struct {
 	// Full X509 name specification (https://golang.org/pkg/crypto/x509/pkix/#Name).
 	Subject *X509Subject
 
+	// +optional
+	RawSubject string `json:"rawSubject,omitempty"`
```

Comment:
This should probably be called `LiteralSubject` instead of `RawSubject` because that would imply DER encoded or something.

Reply:
I didn't think that when I first saw the field, but I could see how someone could easily interpret it that way. Either way, a description would help clear that up.